lms/djangoapps/courseware/tests/test_course_info.py

```diff
@@ -120,7 +120,7 @@ def test_non_live_course(self):
         self.assertRedirects(response, expected_url)
 
     @mock.patch.dict(settings.FEATURES, {'DISABLE_START_DATES': False})
-    @mock.patch("courseware.views.views.strftime_localized")
+    @mock.patch("util.date_utils.strftime_localized")
```

Comment:
I guess it was correct before where the code was, but when I moved it, this test broke until I used the more obvious patch that matched the import.